src/imp.rs

```diff
@@ -251,7 +251,8 @@ fn strike_through_attributes(
             true
         }
     });
-    dec_attrs.extend_from_slice(&strike_attrs[..]);
+
+    strike_attrs.iter().for_each(|attr| dec_attrs.insert(0, attr.clone()));
```

Comment:
I wonder if there's a possible downside to having the on-all-structs attrs before the per-struct, but I don't think there is.

I'd prefer to keep the order of `strikethrough` attrs, though, i.e. use `dec_attrs.splice(0..0, strike_attrs.iter().cloned());` or similar.
(On the struct where the `#[strikethrough[…]]` attribute is declared, it might be nice to even keep the order of strikethrough and non-strikethrough attributes. But that's probably much complexity for little gain.)

Reply:
See latest commit, I've tried to at least only insert the derive attrs first then push the rest at the end.